setup.py

```diff
@@ -10,12 +10,11 @@
     long_description = r.read()
 
 setup(
-    name="sickrage",
-    version="0.0.1",
+    name="medusa",
     description="Automatic Video Library Manager for TV Shows",
     long_description=long_description,
     packages=find_packages(),
-    install_requires=['six', 'requests', 'tornado', 'profilehooks', 'mako', 'pygithub', 'contextlib2', ],
+    install_requires=['six', 'profilehooks', 'contextlib2', ],
```

Comment:
@ratoaq2 @medariox @p0psicles any of you knows what is the use for this setup.py ?